rcongui/src/components/layout/sidebar/About.jsx

```diff
@@ -0,0 +1,138 @@
+      setReleasesStore(storedStore)
+    }
+    loadReleases()
+  }, [])
+
+  const handleClickOpen = () => {
+    setOpen(true)
+  }
+
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  const descriptionElementRef = React.useRef(null)
+  React.useEffect(() => {
+    if (open) {
+      const { current: descriptionElement } = descriptionElementRef
+      if (descriptionElement !== null) {
+        descriptionElement.focus()
+      }
+    }
+  }, [open])
+
+  const latestReleaseVersion = releasesStore.releases?.[0]?.name ?? ''
+  const isUpToDate = latestReleaseVersion.trim() === apiVersion.trim()
+
+  return (
+    <React.Fragment>
+      <ListItem disablePadding onClick={handleClickOpen}>
+        <ListItemButton>
+          <ListItemIcon sx={{ minWidth: 0 }}>
+            <InfoIcon />
+          </ListItemIcon>
+          <ListItemText primary='About' />
+        </ListItemButton>
+      </ListItem>
+      <Dialog
+        open={open}
+        onClose={handleClose}
+        aria-labelledby='about-dialog-title'
+        aria-describedby='about-dialog-description'
+        fullWidth
+        maxWidth='md'
+      >
+        <DialogTitle id='about-dialog-title'>About</DialogTitle>
+        <DialogContent dividers={true}>
+          <Stack direction={['column', 'row']} gap={2} sx={{ mb: 2 }}>
+            <Button
+              size='small'
+              variant='contained'
+              LinkComponent='a'
+              href={siteConfig.documentationUrl}
+              target='_blank'
+              startIcon={<FontAwesomeIcon icon={faBook} />}
+            >
+              Documentation
+            </Button>
+            <Button
+              size='small'
+              variant='contained'
+              LinkComponent='a'
+              href={siteConfig.discordUrl}
+              target='_blank'
+              startIcon={<FontAwesomeIcon icon={faDiscord} />}
+            >
+              Need Help?
+            </Button>
+            <Button
+              size='small'
+              variant='contained'
+              LinkComponent='a'
+              href={siteConfig.githubUrl}
+              target='_blank'
+              startIcon={<FontAwesomeIcon icon={faGithub} />}
+            >
+              Found an issue? Submit a PR!
+            </Button>
+          </Stack>
+          <DialogContentText id='about-dialog-description' ref={descriptionElementRef} tabIndex={-1}>
+            <Typography variant='h5'>Hell Let Loose (HLL) Community RCON (CRCON)</Typography>
+            <Typography>
+              An extended RCON tool for Hell Let Loose, meant to replace the official tool and go WAY beyond.
+            </Typography>
+            <Divider sx={{ my: 2 }} />
+            <Typography variant='body1'>API Version: {isLoading ? 'Loading...' : apiVersion}</Typography>
+            <Typography variant='subtitle2'>
+                {isUpToDate ? 'You are up to date! 🥳' : 'Your app is out of date 💩. You can update now.'}
```

Comment:
wouldn't it make sense to link somewhere, where an "How to update" guide is available? 🤔